container/endpoint.go

```diff
@@ -210,9 +210,9 @@ func buildExportedEndpoints(conn coordclient.Connection, tenantID string, state
 				}
 			}
 			if len(defep.PortList) > 0 {
-				exp.ports = []uint16{}
+				exp.ports = []string{}
```

Comment:
ports is not a good name for this attribute if it doesn't refer to ports anymore